src/IRutils.cpp

```diff
@@ -606,6 +606,16 @@ namespace irutils {
           default:                            return kUnknownStr;
         }
         break;
+      case decode_type_t::HAIER_AC176:
```

Comment:
You need to update `IRac::strToModel()` too.

Reply:
Of course, I'll update it too.